stylesheets/_session_theme.scss

```diff
@@ -6,12 +6,32 @@
   background: var(--color-cell-background);
 }
 
+.module-message__container {
+  .module-message__text {
+    font-size: 14px;
+    line-height: 18px;
+    text-align: start;
+    overflow-wrap: break-word;
+    word-wrap: break-word;
+    word-break: break-word;
+    white-space: pre-wrap;
+
+    a {
+      text-decoration: underline;
+    }
+  }
+}
+
 .module-message {
   position: relative;
   display: inline-flex;
   flex-direction: row;
   align-items: center;
-  max-width: 65%;
+  max-width: 95%;
+
+  @media (min-width: 1200px) {
```

Comment:
What made you pick 1200px? Trial and error

Reply:
well, starting at 1200 the message will only take at most 65% of the available width.

(1200-380)*0.65 = 533.
(380 being the left pane size)

We can have at most 2 x 200px  images to render as attachment, so we have 133px left.
With the some margins between those images, some margins on the right, the :heavy_check_mark: flag of a sent message on the right, and some padding on the left to still have a left and right side (with outgoing and incoming messages), the 1200px appeared to work well :)

Reply:
Awesome, works for me!